docs/vm-images.md

```diff
@@ -0,0 +1,54 @@
+# Supported and Tested VM Images
+
+## HPC CentOS 7 VM Image
+The HPC Toolkit has supported the [HPC CentOS 7 VM Image][hpcimage] as the
+primary VM image for HPC workloads on Google Cloud since it's release. Since the
+[HPC CentOS 7 VM Image][hpcimage] comes pre-tuned for optimal performance on
+typical HPC workloads, it is the default VM image in our modules, unless there
+is specific requirement for a different OS distribution.
+
+Exceptions:
+
+* [DDN-EXAScaler]: The underlying terraform module has a limitted set of
+  supported images [documented][exascalerimages] in the exascaler-cloud-terraform
+  github repo. **This requirement only applies to the servers, not the clients.**
+* [omnia-install]: Only provides support for Rocky 8.
+
+[hpcimage]: https://cloud.google.com/blog/topics/hpc/introducing-hpc-vm-images
+
+## Ubuntu
+The HPC Toolkit supports Ubuntu based VM images in the majority of our modules,
+with a couple exceptions:
+
+* [htcondor-configure]: Only provides support for the HPC CentOS 7 image.
+* [nfs-server]: Only provides support for CentOS 7 images for the server itself.
+* [DDN-EXAScaler]: The underlying terraform module has a limitted set of
+  supported images [documented][exascalerimages] in the exascaler-cloud-terraform
+  github repo. **This requirement only applies to the servers, not the clients.**
+* [omnia-install]: Only provides support for Rocky 8.
+
+HPC Toolkit modules and startup scripts
+have been evaulated against the `ubuntu-2004-lts` image family. For more
+information about the Ubuntu Google Cloud images, see the Canonical
```

Comment:
nit: should Canonical be capitalized?

Reply:
It's the company name that publishes ubuntu: https://canonical.com/

Reply:
Ha. I see.